main.rb

```diff
@@ -0,0 +1,50 @@
+require './app'
+
+def menu
+  puts 'Please choose an option by entering a number: '
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+  user_choice = gets.chomp.to_i
+  puts user_choice
+  execute(user_choice)
+end
+
+# rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
+
```

Comment:
# Optional
- I found that you have disabled the Linter. Note that [Rubocop](https://www.rubocop.org/) is a Ruby static code analyzer (a.k.a. linter) and code formatted. Linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. Kindly do not disable Linter